package.json

```diff
@@ -10,16 +10,16 @@
   },
   "private": true,
   "scripts": {
-    "dev": "webpack --config war/webpack.config.js",
-    "prod": "webpack --config war/webpack.config.js --mode=production",
+    "dev": "webpack --config webpack.config.js",
+    "prod": "webpack --config webpack.config.js --mode=production",
```

Comment:
Part of the Webpack build invoked from the root Maven module, so the build definition moves to the root Maven module.